python/GangaLHCb/Lib/Applications/BenderBox.py

```diff
@@ -0,0 +1,520 @@
+# -*- coding: utf-8 -*-
```

Comment:
> anyhow it does not harm..

Agreed, this is kind of what I was getting at by saying I was guessing that your IDE put it in automatically. As you say it doesn't matter either way, I was attempting to reassure @mesmith75 :wink: